onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -164,6 +168,8 @@ namespace python {
 namespace py = pybind11;
 using namespace onnxruntime;
 using namespace onnxruntime::logging;
+using ProviderOptionsMap = std::map<std::string, std::map<std::string, std::string>>;  
```

Comment:
use unordered_map instead of map where possible. here we don't need order so unordered_map is better.